fsw/src/ds_msgdefs.h

```diff
@@ -70,7 +70,7 @@
  *  \par Command Verification
  *       Evidence of success may be found in the following telemetry:
  *       - #DS_HkPacket_t.CmdAcceptedCounter will reset to zero
- *       - The #DS_RESET_CMD_EID debug event message will be sent
+ *       - The #DS_RESET_INF_EID debug event message will be sent
```

Comment:
```suggestion
 *       - The #DS_RESET_INF_EID informational event message will be sent
```

Reply:
Thanks Justin. I've added the updated comments in now.